modules/cli-manager-adding-plugin-yaml.adoc

```diff
@@ -0,0 +1,67 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/cli_manager/cli-manager-install.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="cli-manager-adding-plugin-yamls_{context}"]
+= Adding a plugin to the {cli-manager}
+
+You can add or update a CLI plugin to the {cli-manager} using the YAML View.
+
+.Prerequisites
+
+* You are logged in to {product-title} as a user with the `cluster-admin` role.
+* The {cli-manager} is installed.
+
+.Procedure
+
+. Log in to the {product-title} web console.
+
+. Navigate to *Operators* -> *Installed Operators*.
+
+. Select the *{cli-manager}* from the list.
+
+. Select the *CLI Plugin* tab.
+
+. Click *Create Plugin*.
+
+. Enter the YAML file with the information for the plugin you are installing in the text box. See the example YAML file below.
++
+.Example YAML file to add a plugin
+
+[source,yaml]
+----
+apiVersion: config.openshift.io/v1alpha1
+kind: Plugin
+metadata:
+  name: <plugin_name> <1>
+spec:
+  shortDescription: <short_description_of_the_plugin_name>
+  description: <long_description_of_the_plugin_name>
+  version: <version> <2>
+  caveats: <3>
+  homepage: <plugin_homepage>
+  platforms:
+  - platform: <platform> <4>
+    image: <plugin_image>
+    imagePullSecret: <5>
+    bin: <plugin_name>
+    files:
+    - from: <plugin_file_path>
+----
+<1> The name of the plugin you will use in commands.
+<2> Version must be in v0.0.0 format.
+<3> Optional: fill in what this is once kasturi tells me.
+<4> Add the architecture for your system. Ex. `linux/amd64`, `darwin/arm64`, `windows/amd64`, or another architecture.
+<5> Optional: If the registry is not public, add a pull secret to access your plugin image.
+
+. Click *Save*.
+
+. Enter the command `oc krew list` to see if the plugin is listed and has been added successfully.
```

Comment:
You'll want to add a code block below for the command instead of saying it here in the step. Here you could say something like, "Verify that the plugin has been added successfully by running the following command" or something similar.

You could also move this to a verification section